src/test/java/seedu/address/logic/parser/CountryNoteEditCommandParserTest.java

```diff
@@ -21,37 +19,39 @@
 
     private final TagNoteMap tagNoteMap = new TagNoteMap();
     private final CountryNoteEditCommandParser parser = new CountryNoteEditCommandParser(tagNoteMap);
+    private final String invalidCommandError = "Invalid command format! \n"
+            + "country note edit: Edits the country note at the given index in the last viewed country note list panel.\n"
+            + "Parameters: INDEX (nt/NOTE_STRING ) (t/TAG)...\n"
+            + "Example: country note edit 1 nt/better government stability in recent months";
+    private final String invalidIndexError = "Index is not a non-zero unsigned integer.";
+    private final String invalidTagError = "Tags names should be alphanumeric and have a maximum of 45 characters";
+    private final String invalidNoteError = "Notes should not be blank";
```

Comment:
would be better / more in line with other tests to change all these Strings to static final and their names to all caps+underscore

Reply:
okie